features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -83,7 +83,12 @@ export function SidebarNavigation() {
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              onClick={() =>
+                window.open(
+                  "mailto:support@prolog-app.com?subject=Support Request: ",
+                  "_blank",
+                )
+              }
```

Comment:
You probably know by now: Alternatively you could use the `MenuItemLink` with the mailto link in the `href`. That would be easier to test. Also a test would have been great 😄